nginx/map.jinja

```diff
@@ -47,7 +50,11 @@
         },
         'core_module_config': {
             'worker_rlimit_nofile': '4096',
-        }
+            'worker_processes':  '1',
+        },
```

Comment:
We set 'worker_processes': '1' here for ubuntu 12.04, do we need to do this for 14.04 and 'Unknown' as well?